src/components/FullscreenGrid/index.tsx

```diff
@@ -18,54 +28,14 @@ export const Header = ({
   );
 };
 
-export type PanelAttribs = 'altSidePanel' | 'mainSidePanel';
-
-type ContentShellProps = {
-  altSidePanel?: AnyReactNode;
-  altSidePanelClass?: string;
-  children?: AnyReactNode;
-  className?: string;
-  mainSidePanel?: AnyReactNode;
-  mainSidePanelClass?: string;
-};
-
-export const ContentShell = ({
-  altSidePanel,
-  altSidePanelClass,
-  children,
-  className,
-  mainSidePanel,
-  mainSidePanelClass,
-}: ContentShellProps) => {
-  return (
-    <React.Fragment>
-      <aside
-        className={makeClassName(styles.mainSidePanel, mainSidePanelClass)}
-      >
-        {mainSidePanel}
-      </aside>
-      <main className={makeClassName(styles.content, className)}>
-        {children}
-      </main>
-      <aside className={makeClassName(styles.altSidePanel, altSidePanelClass)}>
-        {altSidePanel}
-      </aside>
-    </React.Fragment>
-  );
-};
+export const ContentShell = ContentShellComponent;
 
-const FullscreenGrid = ({
-  children,
-  className,
-}: {
-  children?: AnyReactNode;
-  className?: string;
-}) => {
+export const FullscreenGridBase = ({ children, className }: Props) => {
```

Comment:
Out of interest, why did this need to be converted into a class component?

Reply:
It's not a class-based component but I added the `Base` suffix as this is now a connected component. This is needed to retrieve the `fullGrid` reducer data.